instances/latest/contentTypes/application_vnd.nest-simulator.model+cpp.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.om-i.org/props/"
+  },
+  "@id": "https://openminds.om-i.org/instances/contentTypes/application_vnd.nest-simulator.model+cpp",
+  "@type": "https://openminds.om-i.org/types/ContentType",
+  "dataType": null,
```

Comment:
Could we provide this as well? ( Is this code? Might be missing from terms)

Reply:
let's do this later. i would like this information being given for all content types